p2p/src/net/mock/backend.rs

```diff
@@ -646,3 +650,8 @@ where
         }
     }
 }
+
+fn send_resp<T>(response: oneshot::Sender<T>, res: T) {
```

Comment:
I don't think spelling out `send_repsponse` in full would hurt here.

Reply:
Sure, updated.